UTKcataloging/mods_to_rdf
# Add coords and wgs.

What Does this Do?
==================

Adds WGS and splits coords from spatial

How Should This Be Tested?
==========================

Make sure tests pass and changes make sense.

Additional Notes
================

This maps coords to wgs -- not ica.  ICA does not have a coordinates property; it has a coordinates class.  It would only make sense to use that ontology if we were doing something like this:

``` turtle

@prefix ica: <https://www.ica.org/standards/RiC/ontology#Coordinates> .
@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
@prefix skos: <http://www.w3.org/2004/02/skos/core#> .

<https://some-geographic-location> a ica:Coordinates ;
    rdfs:label "House Mountain" ;
    ica:latitude "35.58546" ;
    ica:longitude "-83.84444" ;
    skos:sameAs <https://sws.geonames.org/4630912> .
 

```
